src/devicemanager/devicemanager.cpp

```diff
@@ -147,6 +153,28 @@ DeviceManager* DeviceManager::self()
     return self;
 }
 
+void DeviceManager::doBroadcast()
+{
+    QByteArray datagram = "Discovery";
+    _broadcastSocket.writeDatagram(datagram, QHostAddress::Broadcast, 30303);
```

Comment:
Why we are using port 30303 ?

Reply:
This is as documented in the "Communications Protocol v1.7.pdf"